Expiration and notifications for manually confirmed paid reservations

SanttuA opened the pull request:
Changes:
- Added new email notification type for manually confirmed reservations that are confirmed and ready for payment
- Changed manually confirmed reservations to expire in given hours `RESPA_PAYMENTS_PAYMENT_REQUESTED_WAITING_TIME` or in default 24hrs.
- Manually confirmed reservations which user has started to pay i.e. in `WAITING_FOR_PAYMENT` state are expired as fast as normal paid reservations.

SanttuA committed:
Added requested order expiring and notifications

SanttuA committed:
Changed order expiration to handle new states

Changes:
- Expire only "ready for payment" orders slower in hours/days time.
- Expire "waiting for payment" orders as fast as normal paid reservations

aceViilee reviewed (dismissed):
New notification email template still in works?

SanttuA committed:
Changed when to expire waiting for payment orders

Instead of expiring since staff confirmed, expire since last order modification
